app/scripts/controllers/swaps.js

```diff
@@ -549,21 +556,21 @@ export default class SwapsController {
       });
 
       // The total fee is aggregator/exchange fees plus gas fees.
-      // If the swap is from ETH, subtract the sourceAmount from the total cost.
-      // Otherwise, the total fee is simply trade.value plus gas fees.
-      const ethFee =
-        sourceToken === ETH_SWAPS_TOKEN_OBJECT.address
-          ? conversionUtil(
-              totalWeiCost.minus(sourceAmount, 10), // sourceAmount is in wei
-              {
-                fromCurrency: 'ETH',
-                fromDenomination: 'WEI',
-                toDenomination: 'ETH',
-                fromNumericBase: 'BN',
-                numberOfDecimals: 6,
-              },
-            )
-          : totalEthCost;
+      // If the swap is from the selected chain's default token, subtract
+      // the sourceAmount from the total cost. Otherwise, the total fee
+      // is simply trade.value plus gas fees.
+      const ethFee = isSwapsDefaultTokenAddress(sourceToken, chainId)
```

Comment:
nit: Do we want a more descriptive var name here?  `defaultTokenFee` or `transactionTokenFee`? Just a thought.  Naming is hard though.

Reply:
This is a good point. There actually are a bunch of variables related to swaps with 'eth' in the name that need to be updated. I think "chainCurrency" is a good alternative to "eth". Because the changes are so extensive, I'd like to do this in a separate/future PR, to minimize chance of missing bugs. I have started the work on this branch https://github.com/MetaMask/metamask-extension/pull/new/eth-naming-updates-swaps-alt-nets